src/test/java/com/mercadopago/client/order/OrderClientTest.java

```diff
@@ -153,14 +152,27 @@ void createTransactionWithRequestOptionsSuccess() throws MPException, MPApiExcep
     }
 
     @Test
-    void deleteSuccess() throws MPException, MPApiException, IOException {
-        HttpResponse response = MockHelper.generateHttpResponseFromFile(CREATE_ORDER_RESPONSE_FILE, HttpStatus.OK);
+    void deleteTransactionWithNullIdThrowsException() {
+        String orderId = null;
+        String transactionId = null;
+
+        IllegalArgumentException exception = Assertions.assertThrows(IllegalArgumentException.class, () -> {
+            client.deleteTransaction(orderId, transactionId);
+        });
+
+        Assertions.assertEquals("Order id and Transaction id cannot be null or empty", exception.getMessage());
+    }
+
+    @Test
+    void deleteTransactionSuccessWithValidIds() throws MPException, MPApiException, IOException {
+        HttpResponse response = MockHelper.generateHttpResponseFromFile(DELETE_TRANSACTION_SUCCESS_RESPONSE_FILE, HttpStatus.NO_CONTENT);
```

Comment:
Tem método `generateHttpResponseFromFile` que vc precisa apenas passar o codigo de retorno, quando não há body. O nome do método não condiz com o que ele faz somente... mas pode utiliza-lo para estes casos.
```suggestion
        HttpResponse response = MockHelper.generateHttpResponseFromFile(HttpStatus.NO_CONTENT);
```